superset/assets/cypress.json

```diff
@@ -5,6 +5,7 @@
   "ignoreTestFiles": ["**/!(*.test.js)"],
   "projectId": "fbf96q",
   "defaultCommandTimeout": 10000,
+  "chromeWebSecurity": false,
```

Comment:
maybe sort in abc order? cc @john-bodley ;)

Reply:
sorted now.